tasks1.js

```diff
@@ -0,0 +1,107 @@
+// 1
+function convertMinutesToHours(minutes) {
+  return (Math.floor(minutes / 60)) + ':' + (minutes % 60);
+}
+console.log(convertMinutesToHours(75));
+
+
+// 2
+function isTextUppercase(text) {
+  return /^[A-Z]*$/.test(text)
```

Comment:
Good, can be done easier
```suggestion
  return text.toUpercase() === text;
```